libglusterfs/src/inode.c

```diff
@@ -1353,7 +1339,8 @@ inode_rename(inode_table_t *table, inode_t *srcdir, const char *srcname,
     /* free the old dentry */
     dentry_destroy(dentry);
 
-    inode_table_prune(table);
+    if (table)
```

Comment:
`table` has already been accessed before. We shouldn't check it here.

Doing this check will cause Coverity assume that it can be `NULL`, so the access in previous lines will be flagged as a potential "NULL dereference".

Reply:
Correct, removed.